h/security/acl.py

```diff
@@ -98,15 +101,14 @@ def _for_group(cls, group):
         yield Allow, client_authority_principal, Permission.Group.READ
         yield Allow, client_authority_principal, Permission.Group.MEMBER_READ
         yield Allow, client_authority_principal, Permission.Group.MEMBER_ADD
-        yield Allow, client_authority_principal, Permission.Group.ADMIN
+        yield Allow, client_authority_principal, Permission.Group.EDIT
 
-        # Those with the admin or staff Role should be able to admin/edit any
-        # group
-        yield Allow, Role.STAFF, Permission.Group.ADMIN
-        yield Allow, Role.ADMIN, Permission.Group.ADMIN
+        # Let admins and staff edit the group in the admin page interface
+        yield Allow, Role.STAFF, Permission.AdminPage.GROUPS
+        yield Allow, Role.ADMIN, Permission.AdminPage.GROUPS
```

Comment:
When we reach the promised land of our new permissions system, it won't be so route specific any more. So instead of asking "what can a user do for this context" we will be asking the simpler question of "does the user have Permission.AdminPage.GROUPS?"

This is much easier to answer and means we won't have the duplication we have here between this and the main body of the admin page stuff above.